integration-tests/bats/helper/common.bash

```diff
@@ -35,6 +35,10 @@ current_dolt_user_email() {
 setup_no_dolt_init() {
     export PATH=$PATH:~/go/bin
     cd $BATS_TMPDIR
+    
+    # remove directory if exists
```

Comment:
Might add a comment about why this is necessary (re-runs eventually recycle pids)